virt/backup_restore/virt-backup-restore-snapshots.adoc

```diff
@@ -27,12 +27,14 @@ include::modules/virt-about-vm-snapshots.adoc[leveloffset=+1]
 [id="creating-snapshots_virt-backup-restore-snapshots"]
 == Creating snapshots
 
-You can create snapshots of virtual machines (VMs) by using the {product-title} web console or the command line.
+You can create snapshots of virtual machines (VMs) by using the {product-title} web console or command line. You can also create a snapshot clone by using the {product-title} web console and then use the clone to create a VM.
```

Comment:
This change is not needed, because there's no such thing as a a snapshot clone

Reply:
I removed the sentence you highlighted as being unnecessary, Jenia. Regarding removing the word 'the' in the first sentence, our doc style is to include the article so I left it as is.